pkg/controller/jaeger/elasticsearch.go

```diff
@@ -41,11 +47,11 @@ func (r *ReconcileJaeger) applyElasticsearches(jaeger v1.Jaeger, desired []esv1.
 			"elasticsearch": d.Name,
 			"namespace":     d.Namespace,
 		}).Debug("creating elasticsearch")
-		if err := r.client.Create(context.Background(), &d); err != nil {
-			return err
+		if err := r.client.Create(ctx, &d); err != nil {
+			return tracing.HandleError(err, span)
 		}
-		if err := waitForAvailableElastic(r.client, d); err != nil {
-			return errors.Wrap(err, "elasticsearch cluster didn't get to ready state")
```

Comment:
What about the wrapped error - could either go inside or outside the `tracing.HandleError`.

Reply:
I'll add it back, as it might be adding some value here.